ros2_modulo_control/Dockerfile

```diff
@@ -1,13 +1,12 @@
 ARG BASE_IMAGE=ghcr.io/aica-technology/ros2-modulo
 ARG BASE_TAG=galactic
 
-FROM ${BASE_IMAGE}:${BASE_TAG}
+FROM ${BASE_IMAGE}:galactic as ros2-control-version-galactic
```

Comment:
It seems this would discard all `galactic-devel` images in our final image layer... unfortunately I think you'll need to split the layers for each base tag. To avoid duplication, you can maybe do something along the following lines:

```dockerfile
FROM ghcr.io/aica-technology/ros2-ws:galactic as ros2-control-sources-galactic
USER ${USER}

RUN mkdir -p ${ROS2_WORKSPACE}/src/ros2_control
WORKDIR ${ROS2_WORKSPACE}/src/ros2_control

RUN git clone ...
...


FROM ${BASE_IMAGE}:galactic as ros2-control-version-galactic
COPY --from=ros2-control-sources-galactic ${ROS2_WORKSPACE}/src/ros2_control ${ROS2_WORKSPACE}/src/ros2_control

FROM ${BASE_IMAGE}:galactic-devel as ros2-control-version-galactic-devel
COPY --from=ros2-control-sources-galactic ${ROS2_WORKSPACE}/src/ros2_control ${ROS2_WORKSPACE}/src/ros2_control
```

Reply:
Good catch, thanks!